www/src/components/navigation-mobile.js

```diff
@@ -36,6 +36,7 @@ const MobileNavItem = ({ linkTo, label, icon }) => (
     }}
     getProps={getProps}
     to={linkTo}
+    data-gtm-nav-category="Mobile Footer"
```

Comment:
Are you sure that the tag gets propagated to the underlying `<a>` from the `<Link>`? Maybe do inspect element and check that the tag appears?

Reply:
![image](https://user-images.githubusercontent.com/21114044/83533305-a521b000-a4ac-11ea-8189-1e98b9d5cda2.png)

Here's the successful build on Gatsby Cloud with the a tags getting the data attribute 🙂